scripts/updateTokens/types/TokensJson.ts

```diff
@@ -0,0 +1,26 @@
+export interface RawTokenJSON {
+  name?: string;
+  symbol?: string;
+  address?: string;
+  decimals?: number | string;
+}
+
+export interface ValidatedTokenJSON {
+  name: string;
+  symbol: string;
+  address: string;
+  decimals: number | string;
+  logo: tokenLogo;
+}
+
+export interface tokenLogo {
```

Comment:
@Tbaut i'm in the process i've merging these changes into the luke-124-security branch, and this doesn't let me commit without fixing a pre-commit linting error `paritytech/fether/scripts/updateTokens/types/TokensJson.ts:13:9: 'tokenLogo' was used before it was define`

Reply:
oh weird that I haven't been noticed about that. I'd then simply advise to move the lines starting with `export token interface tokenLogo` toward the top of the file.

Reply:
I think it only showed the linting error because I didn't realise I was using an older version of Node.js.